Google.PowerShell/Compute/GceHealthCheckCmdlets.cs

```diff
@@ -12,8 +12,29 @@ namespace Google.PowerShell.ComputeEngine
     /// Adds a Google Compute Engine health check.
     /// </para>
     /// <para type="descritpion">
-    /// Adds a Google Compute Engine health check. Use this cmdlet to create both http and https health checks.
+    /// Adds a Google Compute Engine health check. Use this cmdlet to create both HTTP and HTTPS health checks.
+    /// https://cloud.google.com/compute/docs/load-balancing/health-checks
     /// </para>
+    /// <example>
+    ///   <para> Adds an HTTP health to project "my-project". </para>
+    ///   <para><code> Add-GceHealthCheck "my-health-check" -Project "my-project" -Http </code></para>
+    /// </example>
+    /// <example>
+    ///   <para> Adds an HTTP health to project "my-project". </para>
+    ///   <para><code> Add-GceHealthCheck "my-health-check" -Project "my-project" -Http </code></para>
```

Comment:
Microsoft's PowerShell examples are prefixed with "`PS C:\> `". This actually helps differentiate which is a command and what is text around it. 

Could you please change this and others to:

///   <para><code>PS C:\> Add-GceHealthCheck "my-health-check" -Project "my-project" -Http </code></para>